ams/server/src/main/java/com/netease/arctic/server/optimizing/plan/PartitionEvaluator.java

```diff
@@ -36,7 +36,7 @@ interface Weight extends Comparable<Weight> {
 
   String getPartition();
 
-  void addFile(IcebergDataFile dataFile, List<IcebergContentFile<?>> deletes);
+  boolean addFile(IcebergDataFile dataFile, List<IcebergContentFile<?>> deletes);
```

Comment:
Could you help to add Java doc for this interface?

Reply:
Sure, I will add come docs.

Reply:
Can you add Java doc for other methods and the class too?

Reply:
Sure, of course.